src/diffusers/schedulers/scheduling_dpmsolver_sde.py

```diff
@@ -394,7 +394,7 @@ def _sigma_to_t(self, sigma, log_sigmas):
         t = t.reshape(sigma.shape)
         return t
 
-    # copied from diffusers.schedulers.scheduling_euler_discrete._convert_to_karras
+    # copied from diffusers.schedulers.scheduling_euler_discrete.EulerDiscreteScheduler._convert_to_karras
```

Comment:
```suggestion
    # Copied from diffusers.schedulers.scheduling_euler_discrete.EulerDiscreteScheduler._convert_to_karras
```

Reply:
I explained this in the PR's first comment. Could you see there?

Reply:
Ah I see what you mean now, ok nevermind! :)